### DISCUSSION TITLE
【需求】手动点击选择英雄天赋

### DISCUSSION THREAD

**Discussion by diaolingzc** [cangzhang/ChampR#235] [Ideas]
腾讯服务器近期更新后，无法显示天赋推荐。虽然 [FAQ](https://github.com/cangzhang/champ-r/wiki/FAQ#tencent-server-v1218-%E6%97%A0%E6%B3%95%E6%98%BE%E7%A4%BA%E5%A4%A9%E8%B5%8B%E6%8E%A8%E8%8D%90) 有提供解决方案，但不排除后期腾讯服务器针对性再次处理。

那么是否可以手动选择英雄后查看对应的天赋推荐，保证软件的可使用性，虽然不是最优体验。相应的，转型为资料类软件，不对LOL进程进行任何修改，规避腾讯服务器的封号规则(第三方软件)。

**Comment by cangzhang** [comment]
有道理，我会在之后实现这个需求

**Comment by rtyyrfe** [comment]
按照FAQ操作后确实能用了，可是第二天再用就失效了，又得重新弄一遍，真的难受

**Comment by cangzhang** [comment]
关掉 wegame, 先打开最新版本 ChampR, 再打开 lol 客户端. 
按照上面步骤操作绝对不需要每次重新转换客户端

**Comment by rtyyrfe** [comment]
> 关掉 wegame, 先打开最新版本 ChampR, 再打开 lol 客户端. 按照上面步骤操作绝对不需要每次重新转换客户端

噢噢，原来是wegame的原因，感谢解答

**Comment by ws34586** [comment]
> 关掉 wegame, 先打开最新版本 ChampR, 再打开 lol 客户端. 按照上面步骤操作绝对不需要每次重新转换客户端

意思是用champr的话，就没法用WeGame的野怪计时之类的功能吗
